Rubberduck.Core/UI/Refactorings/RemoveParameters/RemoveParametersPresenter.cs

```diff
@@ -1,16 +1,11 @@
-﻿using System.Windows.Forms;
+﻿using Rubberduck.Interaction;
 using Rubberduck.Resources;
-using Rubberduck.UI;
-using Rubberduck.UI.Refactorings;
-using Rubberduck.UI.Refactorings.RemoveParameters;
+using Rubberduck.Refactorings.RemoveParameters;
+using System.Windows.Forms;
```

Comment:
Should this reference be here still?

Reply:
the View itself is implemented as an `IRefactoringDialog` which piggy-backs off of Windows Forms' `DialogResult`